app/views/pulitzer/content_elements/edit_multiple.html.erb

```diff
@@ -0,0 +1,25 @@
+<h1>Content elements for <%= @post.title %></h1>
+
+<%= form_tag post_content_elements_update_multiple_path, method: :put, multipart: true do %>
+  <% @content_elements.each do |element| %>
+    <%= fields_for "content_elements[]", element  do |f| %>
```

Comment:
I'd like to convert this to use thin_man instead of fields_for, let's talk about it when we get a chance.